messaging/src/main/scala/uk/ac/wellcome/messaging/worker/monitoring/tracing/ContextCarrier.scala

```diff
@@ -0,0 +1,27 @@
+package uk.ac.wellcome.messaging.worker.monitoring.tracing
+
+import java.util
+
+import io.opentracing.propagation.{Format, TextMapAdapter}
+import io.opentracing.{SpanContext, Tracer}
+
+import scala.collection.JavaConverters._
+
+trait ContextCarrier[T] {
```

Comment:
Can we get some comment docs on some of these traits? Pretty unsure what this one does without any context.